armi/physics/neutronics/crossSectionSettings.py

```diff
@@ -145,12 +145,12 @@ class XSSettings(dict):
     section settings are dependent on user settings.
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, **kwargs) -> None:
```

Comment:
Yeah, I went back and forth on this.

I'm not sure I really care about documenting `__init__` or `__repr__` as part of the API. Though they are both publicly accessible.

Yeah, okay, let's say there is no reason to document the Python-standard-magic methods that are `__whatever__`.  That reduces a lot of overhead, and I think it will leave us just as clear.

Doing it now.